src/com/phonebookservice/util/QueryParam.java

```diff
@@ -0,0 +1,5 @@
+package com.phonebookservice.util;
+
+public abstract class QueryParam {
```

Comment:
Why abstract not interface ?? 

what is the difference between abstract class and interface ? when we use abstract class and when we use interface ?

Reply:
abstract class: contains abstract and non-abstract methods (at least contain 1 abstract method). can also have final, not final, static, and non-static variables. can have a constructor.
Interface class: contains only abstract methods. can only have final and static variables. only public methods. can not have constructors.
use abstract class: if we want to extend subclasses and have a common implementation to add in abstract class.
use interface: if we want each subclass to implement its own implementation